jxp/Animal/lib/Person.pm

```diff
@@ -0,0 +1,151 @@
+package Person;
+
+use 5.006;
+use strict;
+use warnings;
+use parent qw(LivingCreature);
+
+=head1 NAME
+
+Person - The great new Person!
+
+=head1 VERSION
+
+Version 0.01
+
+=cut
+
+our $VERSION = '0.01';
+
+
+=head1 SYNOPSIS
+
+Quick summary of what the module does.
+
+Perhaps a little code snippet.
+
+    use Person;
+
+    my $foo = Person->new();
+    ...
+
+=head1 EXPORT
+
+A list of functions that can be exported.  You can delete this section
+if you don't export anything, such as for a purely object-oriented module.
+
+=head1 SUBROUTINES/METHODS
+
+=head2 function1
+
+=cut
+
+sub sound{ "humming" }
+
+=head2 function2
+
+=cut
+
+sub speak {
+  my $class = shift;
+  my $message = shift;
+  
+  if ( defined $message ){
```

Comment:
is this code repeated in Animal?

Could this logic have been moved to LivingCreature?